starknet_py/tests/e2e/tests_on_networks/trace_api_test.py

```diff
@@ -22,48 +24,46 @@
 
 
 @pytest.mark.asyncio
-async def test_trace_transaction(client_goerli_integration):
+async def test_trace_transaction(client_sepolia_integration):
```

Comment:
Should we create a block with all of those transactions? do we need it?

Reply:
We can try to find a block that has many different transaction types, not necessarily all the possible types. It should be easier to find on testnet.

Reply:
I found block with various transactions on integration. I have all this 
```
        InvokeTransactionV1, InvokeTransactionV3, DeployAccountTransactionV1,L1HandlerTransaction
```
It's not easy to find sth better on Testnet or I'm not able to do that :)